gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-checker = "3.45.0"
+checker = "3.47.0"
```

Comment:
3.48.3 is the most recent; but since 3.48.0 it's failing on 

```
> Task :backend:impl:compileJava
warning: The following options were not recognized by any processor: '[SubtypingChecker_qualDirs]'
error: warnings found and -Werror specified
```